simple_graph/test_wgraph.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+import weighted_graph as wgraph
+from collections import OrderedDict
+
+
+def test___init__():
+    """Test instantiates simple graph with dictionary."""
+    assert wgraph.Wgraph().dict == {}
+
+
+def test_nodes_empty(empty_graph):
+    """Test return list of all nodes in an empty graph"""
+    assert empty_graph.nodes() == []
+
+
+def test_nodes_populated(populated_graph):
+    """Test return list of all nodes in a populated graph"""
+    l = ['a', 'b', 'c', 'd', 'e']
+    sortedlist = populated_graph.nodes()
+    sortedlist.sort()
+    assert sortedlist == l
+
+
+def test_edges(populated_graph):
+    """Test return a list of all edges in the graph"""
+    edges = [('a', 'b'), ('b', 'c'), ('c', 'b'),
+             ('c', 'a'), ('b', 'd'), ('d', 'c')]
+    for edge in edges:
+        assert edge in populated_graph.edges()
+
+
+def test_add_node_empty(empty_graph):
+    """Test adds a new node 'n' to the empty graph"""
+    empty_graph.add_node('a')
+    assert empty_graph.has_node('a')
```

Comment:
It's a bit self-referential to use a method of the graph to test the result of another method of the graph.  Might be better to do something like

```python
assert 'a' in emtpy_graph.dict
```